### PR TITLE
chore: enable arm64

### PR DESCRIPTION
**Pull request by s22chan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

see https://conda-forge.org/blog/2020/10/29/macos-arm64/

### CODE REVIEW

**Comment by s22chan** [comment]
@conda-forge-admin, please rerender

**Comment by s22chan** [comment]
hmm it didn't trigger the arm builds

**Comment by ovesh** [comment]
Relevant docs: https://conda-forge.org/docs/maintainer/knowledge_base/#how-to-enable-cross-compilation

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,13 +9,17 @@ source:
...
+  number: 1
   script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
   skip: true  # [py>310 or py<38]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```
[the remark this answers is not in the document]
Comment (s22chan) [reply recipe/meta.yaml]: cython wasn't thanks.  the others is because it's cross compiling

**Review by ovesh — APPROVED** [review]

**Pull request merged**